instrumentation/trilogy/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History: opentelemetry-instrumentation-trilogy
 
+### v0.51.2 / 2023-xx-xx
+
+* Ensure SQL obfuscation can handle invalid byte encoding. [#345](https://github.com/open-telemetry/opentelemetry-ruby-contrib/pull/345)
+
```

Comment:
```suggestion
```

The release scripts dynamically generate release notes based on PR titles that follow conventional commits format. https://www.conventionalcommits.org/en/v1.0.0/

Please rename this PR's title to use conventional commits.

Reply:
Neat, TIL about conventional commits! 😮 I've dropped the CHANGELOG commit and updated the PR description.